RELEASE-NOTES.txt

```diff
@@ -4,6 +4,7 @@
 -----
 
 * [*] [internal] Block editor: Upgrade Android 13 [https://github.com/wordpress-mobile/WordPress-Android/pull/18477]
+* [***] [Jetpack-only] Plans: Bringing WPCOM plans to JP app, behind a feature flag [https://github.com/wordpress-mobile/WordPress-Android/issues/18417]
```

Comment:
Do we need to explicitly mention that it's under a feature flag?

Reply:
I think we should add `[internal]` to the beginning instead of mentioning the feature flag. 
You can check the meaning of `[internal]`,
**Internal ref:** pdcxQM-Km-p2

Reply:
> Do we need to explicitly mention that it's under a feature flag?

We were mentioning this explicitly during Jetpack migration.  Probably, not necessary here.

> add [internal] to the beginning instead of mentioning the feature flag

[internal] if a feature is not going to be released, but needs testing.  it's going to be released right.